app/config/doctrine.yml

```diff
@@ -20,6 +20,8 @@ doctrine:
         types:
             enum_bool: Common\Doctrine\Type\EnumBoolType
             locale: Common\Doctrine\Type\LocaleType
+            datetime: Common\Doctrine\Type\UTCDateTimeType
+            datetimetz: Common\Doctrine\Type\UTCDateTimeType
```

Comment:
You are overwriting "datetime" and "datimetz", but why not "time"?
http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/known-vendor-issues.html